src/Select/index.stories.js

```diff
@@ -294,6 +294,10 @@ InAForm.parameters = {
 
 export const Controlled = () => {
   const [value, setValue] = useState(null);
+  const [items, setItems] = useState([
+    { value: "checking1234", label: "Checking (1234)" },
+    { value: "savings4321", label: "Savings (4321)" },
```

Comment:
I updated the controlled examples in both components to use a state var instead - this is much closer to real world usage